public/app/plugins/datasource/loki/components/monaco-query-field/monaco-completion-provider/completions.ts

```diff
@@ -0,0 +1,241 @@
+import { AGGREGATION_OPERATORS, RANGE_VEC_FUNCTIONS } from '../../../syntax';
```

Comment:
These are the functions that, based in a given situation, select which completion options to return.